src/front-end/src/components/MenteeForm/MenteeForm.test.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import userEvent from '@testing-library/user-event';
+import { act } from 'react-dom/test-utils';
+import { mockWindowMatchMedia } from '../../helper-methods';
+
+import MenteeForm from './MenteeForm';
+
+let container = null;
+
+mockWindowMatchMedia();
+
+beforeEach(() => {
+  container = document.createElement('div');
+  document.body.appendChild(container);
+});
+
+afterEach(() => {
+  // cleanup on exiting
+  unmountComponentAtNode(container);
+  container.remove();
+  container = null;
+});
+
+it('renders without crashing', () => {
+  render(<MenteeForm />, container);
+  expect(container.textContent).toBe(
+    'Sign up as a MenteeNameEmailTime-zoneSelect timezone..Top 3 skills\xa0DescriptionSubmit',
+  );
+});
+
+test('renders the correct header text', () => {
+  render(<MenteeForm />, container);
+  const h1 = document.getElementsByTagName('h1');
+  expect(h1[0].textContent).toBe('Sign up as a Mentee');
+});
+
+test('renders the correct input labels', () => {
+  render(<MenteeForm />, container);
+  const getLabels = document.getElementsByTagName('label');
```

Comment:
Generally speaking, I wouldn't include a verb in a variable name. This should be `labels` instead of `getLabels`, IMO. 

Edit: I get why you put getLabels. I think you're trying to express in the var name that those are the labels that got rendered rather than the ones we expect, right? Im that case, I'd use an adjective or a past-tense-verb, like maybe `receivedLabels`, `obtainedLabels`, or `renderedLabels`.